bundle/src/main/java/com/adobe/acs/commons/indesign/dynamicdeckdynamo/models/DynamicDeckInitiatorPageModel.java

```diff
@@ -0,0 +1,154 @@
+            description = "Enter a InDesign deck name. e.g. summer-2020",
+            hint = "sample")
+    private String deckTitle;
+
+    @Inject
+    @Named(value = "destinationPath")
+    @FormField(
+            name = "Deck Destination Folder Path",
+            description = "Select the folder path where the InDesign deck will be generated.",
+            required = true,
+            component = PathfieldComponent.FolderSelectComponent.class,
+            options = {"base=/content/dam"},
+            hint = "/content/dam/someFolder",
+            category = "General")
+    private String destinationPath;
+
+    @Inject
+    @Named(value = "masterAssetPath")
+    @FormField(
+            name = "Master Asset Path",
+            description = "Select the master asset. InDesign deck's header & footer info will be taken from master asset's metadata",
+            component = PathfieldComponent.AssetSelectComponent.class,
+            options = {"base=/content/dam"},
+            hint = "/content/dam/someAsset",
+            category = "General")
+    private String masterAssetPath;
+
+    @Inject
+    @Named(value = "templatePath")
+    @FormField(
+            name = "Template Folder Path",
+            description = "Select the InDesign template folder for generating the InDesign deck. Make sure that the folder must have a template file and the mapped XML file in the same folder.",
+            required = true,
+            component = PathfieldComponent.FolderSelectComponent.class,
+            options = {"base=/content/dam"},
+            hint = "/content/dam/someFolder",
+            category = "General")
+    private String templatePath;
+
+    public enum Mode {
+        COLLECTION, QUERY, TAGS
+    }
+
+    @Inject
+    @Named(value = "operationMode")
+    @FormField(
+            name = "Operation Mode",
+            description = "Select the operation mode to supply the assets to the deck dynamo.",
+            required = false,
+            component = RadioComponent.EnumerationSelector.class,
+            options = {"horizontal", "default=COLLECTION"})
+    private Mode operationMode;
+
+    @Inject
+    @Named(value = "collectionPath")
+    @FormField(
+            name = "Collection",
+            description = "Select the collection from the drop-down below.",
+            component = CollectionSelectComponent.class,
+            category = "General")
+    private String collectionPath;
+
+    @Inject
+    @Named(value = "assetQuery")
+    @FormField(
+            name = "Query to fetch assets",
+            component = GenericListSelectComponent.class,
+            category = "General",
+            description = "Select the query from the drop-down below. These queries are listed from the 'Query List' page linked on top-right of this page.",
+            options = {GenericListSelectComponent.GENERIC_LIST_PATH + "=/etc/acs-commons/lists/dynamic-deck-query-list"})
+    private String assetQuery;
+
+    @Inject
```

Comment:
Can we use an explicit Injector here? Ex. `@ValueMapValue` if it comes from the resource property?